Source/Plugin.BLE.Android/GattCallback.cs

```diff
@@ -174,6 +176,14 @@ public override void OnReliableWriteCompleted(BluetoothGatt gatt, GattStatus sta
             Trace.Message("OnReliableWriteCompleted: {0}", status);
         }
 
+        public override void OnMtuChanged(BluetoothGatt gatt, int mtu, GattStatus status)
+        {
```

Comment:
Because this is based on the current master and not on 1.3.0 we will need to also check device ID here. I.e the gatt callback instance is shared among all connected devices and so we need a way to identify the correct device. 
Maybe you can base your PR on the 1.3.0 and that we we will avoid this issue. (FYI 1.3.0 does some gatt callback refactoring in the sense that we have a callback per device instead of a global one). What do you think?